guides/source/active_record_migrations.md

```diff
@@ -827,7 +1036,8 @@ saying that it can't be done.
 
```

Comment:
Above it says: In such cases, you can raise `ActiveRecord::IrreversibleMigration` in your
`down` block.

How would you do that?